withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -603,7 +623,12 @@ PROCEDURE VALIDATEACCOUNT
           DO
            IF ANYWARNING(SHARE,TMPLOOP) AND
               WARNINGSSHARE(TMPLOOP)=TRUE THEN
-            VALIDATESTATUS=VALIDATESLWARNINGFOUND
+            DO
+             VALIDATESTATUS=VALIDATESLWARNINGFOUND
+             IF INVSLWARNSFOUND<>"" THEN
+              INVSLWARNSFOUND=INVSLWARNSFOUND+","
```

Comment:
Same as for comment pertaining to INVACCTWARNSFOUND above.

Reply:
I put in some debug code for the "Loan Warning Found list" and got the output for the longest Possible message 128 characters after the NEWLINE and before any other NEWLINE Command.

Am I good to go?

Here is that output
&quot;errorCode&quot;:503
,&quot;loggingErrorMessage&quot;:&quot;Loan Warning Found List 002,003,004,005,006,007,008,009,010,011,012,013,014,015,016,017,018,019,020,021&quot;
,LENGTH OF PREVIOUS loggingErrorMessage is            128
,&quot;displayErrorMessage&quot;:[&quot;This share/loan is not eligible for check withdrawal.&quot;,
&quot;Please contact the credit union at 800-111-2222 with questions.&quot;
]
Here is the debug code:
 NEWLINE

 PRINT ","
 JSONPROPNAME="loggingErrorMessage"
 CALL JSONPROP
 PRINT QQ+BNOERRMESSAGE+QQ
 NEWLINE
[JPK TEMP DEBUG]
[PRINT QQ+JSONPROPNAME+QQ+":"]
MESSAGELENGTHTOCHECK=","+QQ+JSONPROPNAME+QQ+":"+QQ+BNOERRMESSAGE+QQ
IF BNOERRCODE=BNOERRCODENOTFOUND THEN
 DO
  PRINT ","
  PRINT "LENGTH OF PREVIOUS loggingErrorMessage is "+FORMAT("#############9",LENGTH(MESSAGELENGTHTOCHECK))
  NEWLINE
 END
[END JPK TEMP DEBUG]

Here is the Data Dog log:
https://us5.datadoghq.com/logs?query=%40symxchange.username%3ABNO9000%20%40poweron.file%3ABANNO.CHECK.WITHDRAW.V1.POW%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAY_q9nA2bi4lFAAAAAAAAAAYAAAAAEFZX3E5b0JWQUFESUQyWG1CTUdoNndBQQAAACQAAAAAMDE4ZmVhZjYtYTYyYS00N2ZlLTg0YTYtMzk4OTg3ZmI0ZjQ5&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1717442111084&to_ts=1717614911084&live=true

Reply:
Checked & approved